browser-client.cpp

```diff
@@ -679,6 +679,13 @@ void BrowserClient::OnLoadEnd(CefRefPtr<CefBrowser>, CefRefPtr<CefFrame> frame,
 
 		frame->ExecuteJavaScript(script, "", 0);
 	}
+
+	if (bs->javascript_active) {
+		std::string javascript;
+		javascript = bs->javascript;
+		if (!bs->javascript.empty())
+			frame->ExecuteJavaScript(javascript, "", 0);
```

Comment:
At a glance, I don't understand the switching between `bs->javascript` and `javascript` here.  Why not:
```
if (!bs->javascript.empty())
	frame->ExecuteJavaScript(bs->javascript, "", 0);
```

If there's some sort of requirement here that `bs->javascript` not be accessed in certain conditions (e.g., with ExecuteJavaScript), then why not:
```
std::string javascript = bs->javascript;
if (!javascript.empty())
	frame->ExecuteJavaScript(javascript, "", 0);
```

Reply:
I think this was legacy, or some attempt of me trying to get a log-able string out of the bs-> variable. I simplified it even more to:
```
	if (bs->javascript_active && !bs->javascript.empty())
		frame->ExecuteJavaScript(bs->javascript, "", 0);
	}

```
(for sanity reasons I should probabbly also check `&& bs->javascript != null`, but the string should actually always be initialized at least)